mslib/msui/qt5/ui_add_project_dialog.py

```diff
@@ -55,6 +55,16 @@ def retranslateUi(self, addProjectDialog):
         self.path.setPlaceholderText(_translate("addProjectDialog", "Project Name (No spaces or special characters)"))
         self.description.setPlaceholderText(_translate("addProjectDialog", "Project Descriptions"))
         self.browse.setText(_translate("addProjectDialog", "browse..."))
-        self.label_3.setText(_translate("addProjectDialog", "Choose FTML File (Optional)"))
+        self.label_3.setText(_translate("addProjectDialog", "Choose Flighttrack File (Optional)"))
         self.label_2.setText(_translate("addProjectDialog", "Description"))
         self.selectedFile.setPlaceholderText(_translate("addProjectDialog", "(use browse to pick a file)"))
+
+
+if __name__ == "__main__":
```

Comment:
there is likly an option to get this not in to the file by the conversion

Reply:
I'll look into it.